### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by ashneverdawn**
Resolves: https://github.com/pendulum-chain/pendulum-ink-wrapper/issues/4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
+## Build
 
-cargo install cargo-contract --version 2.1.0
+`cargo contract build`
 
-Contracts
+## Deploy
 
-If you have cargo-contract (https://github.com/paritytech/cargo-contract) installed, you can easily build with:
+UI to deploy and use: https://contracts-ui.substrate.io/?rpc=ws://127.0.0.1:9944
```
Comment (ebma) [inline README.md:15]: Maybe we should change the linked web socket to be one of Foucoco's?
Or we mention multiple web sockets that we 'recommend' in a code box below. I think it's important to link to Foucoco's here.

File: price_feed/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# price_feed
+
+a contract that wraps the dia chain extension which provides a price feed for various assets.
```
Comment (ebma) [inline price_feed/README.md:3]:
```suggestion
This is a contract that wraps the dia chain extension which provides a price feed for various assets.
```

**Review by ebma — COMMENTED** [review]
I'd capitalize all usages of 'pendulum' to 'Pendulum' and use 'ink!' instead of just 'ink'. But that's just nitpicking.

File: README.md (hunk 1 above)
Comment (ashneverdawn) [reply README.md]: Done 👍

**Review by ebma — APPROVED** [review]
LGTM, good job @ashneverdawn 👍

**Pull request merged**